lib/pdf/core/embedded_files.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module PDF
+  module Core
+    module EmbeddedFiles #:nodoc:
+      # The maximum number of children to fit into a single node in the
+      # EmbeddedFiles tree.
+      NAME_TREE_CHILDREN_LIMIT = 20 #:nodoc:
+
+      # The EmbeddedFiles name tree in the Name dictionary (see
+      # Prawn::Document::Internal#names). This name tree is used to store named
+      # embedded files (PDF spec 3.10.3). (For more on name trees, see section
+      # 3.8.4 in the PDF spec.)
+      #
+      def embedded_files
+        names.data[:EmbeddedFiles] ||= ref!(
```

Comment:
`EmbeddedFiles` is available starting PDF 1.4. Document format version should be bumped to the appropriate version.

Reply:
You're right. Fixed